changelog/2926.trivial.rst

```diff
@@ -0,0 +1 @@
+User can access the Helioviewer API using fewer keywords. Either by `observatory` and `measurement` or `instrument` and `measurement`.
```

Comment:
I would expand upon this change log, maybe with a short example.

Reply:
Example showing that we can access the source ID by just using two keywords?

Reply:
Yeah, something like that. The changelog to explain this change better since it's unclear to me right now.